client/src/components/admin/AdminCompetitors/AdminCompetitors.jsx

```diff
@@ -50,11 +50,22 @@ function AdminCompetitors() {
   if (loading && !data) return <Loading />;
   if (error) return <Error error={error} />;
   const { competition } = data;
+  const participatedCompetitors = competition.competitors.filter((competitor) =>
+    competition.competitionEvents.some((competitionEvent) =>
+      competitionEvent.rounds.some((round) =>
+        round.results.some(
+          (result) =>
+            result.person.id === competitor.id && result.attempts.length > 0
+        )
+      )
+    )
+  ).length;
 
   return (
     <>
       <Typography variant="h5" gutterBottom>
-        Competitors
+        Competitors ({participatedCompetitors} /{" "}
+        {competition.competitors.length})
       </Typography>
```

Comment:
This maybe be a bit misleading, if the number is here I would expect it to be the same as the number of competitors in the table.

Instead, I suggest we put this information below the table. I'd show something like "Total: {...}" and "Competed: {...}" :)